configure.py

```diff
@@ -92,6 +92,28 @@ def get_shared_lib_name():
     return namespec[1][3:]
 
 
+def get_tf_version():
+  """
+  Get Tensorflow version as a 4 digits string.
+
+  For example:
+    1.15.2 get 1152
+    2.4.1 get 2041
+
+  The 4-digits-string will be passed to C macro to discriminate different
+  Tensorflow versions. 
+
+  We assume that major version has 1 digits minor version has 2 digits. And
```

Comment:
We assume that major version has 1 digits, minor version has 2 digits and patch version has 1 digits.